system_metrics_collector/README.md

```diff
@@ -1,11 +1,33 @@
-# System metric collector
+# System metrics collector
 
-## Parameters
+## Supported targets
+The system metrics collector package supports the following:
+
+* ROS Distro
+  * ROS 2: `eloquent`
+* OS: `Ubuntu Bionic`
+
+## Description
+This package aims to integrate lightweight aggregation tools in order to collect, measure, and publish aggregate metrics.
 
+The [moving average](https://github.com/ros-tooling/system_metrics_collector/blob/master/system_metrics_collector/src/moving_average_statistics/moving_average.hpp)
+tools provide constant time sample aggregation to produce average, min, max, standard deviation, and sample count. The
```

Comment:
Nit: new line after period and use relative paths for files.

Reply:
Done